Assignment-5/app/src/components/ModifiedDataFetchingAxious.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from "react";
+import axios from 'axios'
+
+function ModifiedDataFetchingAxios() {
+  const [data, setData] = useState([]);
+  const [loading, setLoading] = useState(true);
+
```

Comment:
This task is not working properly if i click on refresh button it updates the data 2 two times on a single click